docs/site/content/docs/latest/tanzu-diagnostics.md

```diff
@@ -113,3 +119,22 @@ This command will attempt to collect diagnostics data for any boostrap cluster t
 ```sh
 tanzu diagnostics collect --bootstrap-cluster-skip --management-cluster-skip --workload-cluster-name=<WORKLOAD_CLUSTER_NAME>
 ```
+
+## Collecting unmanaged cluster diagnostics
+
+Finally, the plugin also supports collecting diagnostics from unmanaged-clusters, a lite weight, local deployment model.
+Because unmanaged-clusters do _not_ have a bootstrap cluster and are _not_ managed by a tanzu management cluster,
+you must provide the name of the unmanaged cluster and the context:
+
+```sh
+tanzu diagnostics collect --unmanaged-cluster-name kind-my-unmanaged-cluster --unmanaged-cluster-context kind-my-unmanaged-cluster
```

Comment:
Do we really need both? I would have thought just one or the other would be enough for us to find the cluster.

Reply:
If so, this conflicts with what was added in the README doc.

Reply:
You do need both.

The diagnostics plugin defaults to using a `<cluster-name>-admin@<cluster-name>` context. I believe this is in support of working out of the box with managed-clusters.

So if you don't provide a context name for unmanaged-cluster, it fails since the kind context names it `kind-<name>`

I'll update both docs on this to reflect it correctly.